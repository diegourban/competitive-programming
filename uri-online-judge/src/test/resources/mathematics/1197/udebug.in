0 0
5 12
2 10
10 10
2 5
1 1
1 22