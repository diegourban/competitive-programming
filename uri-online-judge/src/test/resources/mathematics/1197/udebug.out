0
120
40
200
20
2
44
